index.js

```diff
@@ -1,24 +1,21 @@
 var sep = require('path').sep
-var last = require('lodash').last
 var toString = require('mdast-util-to-string')
 
 module.exports = checkTitle
 
-function checkTitle (remark, opts) {
+function checkTitle (opts) {
+  var title = (opts || {}).title
   return function checkTitleTransformer (root, file) {
+    var filePath = (file.dirname === '.' ? file.cwd : file.dirname) || title || process.cwd();
+    var dirnames = filePath.split(sep)
+    var folder = dirnames[dirnames.length - 1].toLowerCase()
```

Comment:
Ideally, I was hoping to not just have the `folder` as the name, but also the option of sending in a title through an option, or getting it through the git repo of that directory. Hence not using `folder` as the var. Not sure this is worth changing, just noting.